README.md

```diff
@@ -322,3 +323,19 @@ contribute.
 -   See our [code of conduct](CODE_OF_CONDUCT.md) and
     [contributing guidelines](CONTRIBUTING.md) for information about the Carbon
     development community.
+
+### Contributing
+
+You can also directly:
+
+-   [Contribute the language design](CONTRIBUTING.md#contributing-to-the-language-design):
+    feedback on design, new design proposal
+-   [Contribute to the language implementation](CONTRIBUTING.md#contributing-to-the-language-implementation)
+    -   [Carbon Explorer](/explorer/): bug report, bug fix, language feature
+        implementation
+    -   [Carbon Toolchain](/toolchain/), and project infrastructure
+
+You can refer to [`CONTRIBUTING`](CONTRIBUTING.md) for more information, check
+["good first issues"](https://github.com/carbon-language/carbon-lang/labels/good%20first%20issue),
+or join the `#contributing-help` channel on
+[Discord](https://discord.gg/ZjVdShJDAs).
```

Comment:
Slight structure / formatting tweaks, WDYT?

```suggestion
You can **check out some
["good first issues"](https://github.com/carbon-language/carbon-lang/labels/good%20first%20issue)**,
or join the `#contributing-help` channel on
[Discord](https://discord.gg/ZjVdShJDAs). See our full
[`CONTRIBUTING`](CONTRIBUTING.md) documentation for more details.
```

Reply:
Yep, works for me!